### PR TITLE
update tests for v 7.0.0

### PR DESCRIPTION
**Pull request by JorgeG94**

### CODE REVIEW

File: dft-libxc/parallel/functionals_dft/rdft-CNm-HCTH-A-gradient.json [hunk 1]
```diff
@@ -7,126 +7,126 @@
       "value": "1.0000000000",
       "type": "float",
       "precision": 10,
-      "tolerance": 7.5e-07
+      "tolerance": 5e-08
```
Comment (friedc) [inline dft-libxc/parallel/functionals_dft/rdft-CNm-HCTH-A-gradient.json]: I don't think that the tolerances should be tightened.

Comment (JorgeG94) [reply dft-libxc/parallel/functionals_dft/rdft-CNm-HCTH-A-gradient.json]: Will fix!

Comment (JorgeG94) [reply dft-libxc/parallel/functionals_dft/rdft-CNm-HCTH-A-gradient.json]: addressed in 58d37c75a819e0cd78ca205b47f649cc09166a0f

**Review by foxtran — APPROVED** [review]
LGTM! Thank you! As I can see, only HCTH-A and REVSCAN are affected.

**Pull request merged**